internal/apply/apply.go

```diff
@@ -33,6 +35,13 @@ Only a single environment may be suitable for an initial test, but for a real sy
 		environments = promptEnvironments()
 	}
 
+	flog.Infof(":tada: checking project %s. Please use the zero-project.yml file to modify the project as needed.", projectConfig.Name)
```

Comment:
maybe `magnifying` instead of `tada` and remove the part about zero-project since it should appear later?